playground/mocks/config/responseConfig.js

```diff
@@ -62,6 +62,8 @@ const idx = {
     // 'authenticator-verification-select-authenticator',
     // 'authenticator-verification-select-authenticator-ov-m2',
     // 'authenticator-verification-webauthn',
+    // 'authenticator-verification-yubikey',
```

Comment:
You shouldn't (typically) commit changes to the `responseConfig.js` since it affects all developers' default playground experience. This is a DX pain-point we plan to address in the future.